### DISCUSSION TITLE
bluetooth going crazy in a Fossibot

### DISCUSSION THREAD

**Discussion by elibraver** [j-kaltes/Juggluco#146] [Q&A]
hi, im trying to set up juglucco in a fossibot F101 Pro, when i scan the sensor with NFC it recognizes it, but then the bluetooth starts going on and off every second, making it unable to keep any connection. has anybody had a similar issue? i tried rebooting the phone, reseting it to factory settings, if i scan the same sensor with another phone (xiaomi) it works fine. 
 thank you very much

https://github.com/j-kaltes/Juggluco/assets/50751619/b67bc50c-beb5-48f2-9e0a-f6f35d726827

**Comment by j-kaltes** [comment]
Juggluco doesn't change the Android Bluetooth off or on setting. If something other than yourself turns of Bluetooth, it isn't Juggluco. As far as I know Juggluco doesn't have permission to do that.
Did you turn on a Bluetooth Watchdog in xDrip or Android Aps?
Does your phone different with Juggluco turned off?
Consider a factory reset or a new phone. It isn't a very good one: https://www.devicespecifications.com/en/model-cpu/ae965e72

**Comment by elibraver** [comment]
I installed an older version of juggluco and it worked. Version 7.5 works
fine. so I guess there is a bug somewhere in the new version, or it's not
compatible with the phone.
Thank you very much for your answer.

**Comment by j-kaltes** [comment]
Juggluco can not do that. It has target version Android 13 and than never can turn off Bluetooth.
